202403_crop_hail_damage_footprint/get_per_gridcell_data.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Mar 30 14:00:01 2023
+
+Get damage, and exposure information at hazard centroid for each date. 
```

Comment:
In my understanding, more appropriate would be the following: 
```suggestion
Load exposure and damage data at 1km resolution. Get hazard information at exposure centroids for each date and each resolution.  
```